test/data/config-sarif-report/report.sarif

```diff
@@ -9,87 +9,6 @@
           "informationUri": "https://github.com/aquasecurity/trivy",
           "name": "Trivy",
           "rules": [
-            {
-              "id": "AVD-AWS-0086",
-              "name": "Misconfiguration",
-              "shortDescription": {
-                "text": "S3 Access block should block public ACL"
-              },
```

Comment:
does this make sense to you @nikpivkin? I understand that the Go rule was deprecated but should the Rego rule in it's place still pick this up?